modules/nf-core/bismark/align/main.nf

```diff
@@ -9,7 +9,7 @@ process BISMARK_ALIGN {
 
     input:
     tuple val(meta), path(reads)
-    tuple val(meta2), path(fasta)
+    tuple val(meta2), path(fasta, stageAs: 'tmp/*')
```

Comment:
I suggest adding a comment here to explain the situation, and why this is necessary.